### ISSUE TITLE
Unapprove Handles for Past Prospectives

### ISSUE THREAD

**Issue by rangerthreepio** [burningmantech/ranger-clubhouse-web#168]
In order to better clean up prospectives who drop out during the season, it would be great if changing Status to Past Prospective would automatically change Callsign Approved to "Not Approved." There is also an argument to be made that this should happen whenever Status is changed to Auditor, because if someone is being moved from anything to auditor, we just want to double check that they don't carry their approved handle with them.

**Comment by mikeburg** [comment]
So the callsign would be disapproved automatically - should the callsign also be converted to an auditor callsign, and the original placed into the FKA?

**Comment by rangerthreepio** [comment]
VCs need to discuss, but yes, I believe it would be helpful to move callsign to FKA, and autofill LastFYY(NR) (I dunno when NR became the standard, is that a thing?)

**Comment by mikeburg** [comment]
The (NR) suffix feels like hack to identify non-rangers  before the auditor, prospective, and related statuses came online.

The discussion might be worth having to determine if the NR suffix is still needed in the current age.

**Comment by rangerthreepio** [comment]
We didnt do it for the last few years and suddenly people started putting
it back in this year. I'm confuzed.

**Comment by plapsley** [comment]
YF: It is a hack, but it's a useful hack. By adding the (NR) to the callsign, it allows non-Rangers to instantly be identified no matter what Clubhouse report their names appear on.  If you don't have the (NR) suffix, then you need to make sure that every report either (1) excludes auditors or (2) includes their status next to their name.  If we're sure that these two conditions are met (they didn't used to be), then I'm fine with removing the NR thing.

3po: I'm not sure what you mean when you say "We didn't do it for the last few years."  Maybe I'm misremembering but the auditor code has always done that, I think.  We started doing (NR) in 2013.

**Comment by rangerthreepio** [comment]
the auditor code has done it, but Past Prospectives have not been so
handled. I agree with you that your logic above states that we SHOULD have,
but we did not, i think.

**Comment by wsanchez** [comment]
I'd lobby for fixing up the reports if needed.

Status should be reflected in one place in the database, not two, and name fields should contain names, not status info.

It's not the worst thing in the world, but it's the sort of thing that eventually becomes a regret.

**Comment by plapsley** [comment]
Sure, that's unquestionably the right answer.  I'm just saying, let's not stop with (NR) until such time as someone has gone through and made sure the reports (or other places where a Ranger callsign appears) either show status or are gated by status.

**Comment by plapsley** [comment]
New issue #172

**Comment by rangerthreepio** [comment]
To clarify the VC perspective on this: we can keep using it for Auditors
(and I think we have been this year, no problem), but I think we/the
clubhouse aren't using it consistently to handle Past PNVs, and we
obviously don't want to contribute to discord there.

There is likely a lot of housekeeping on Past PNVs around this. This year,
it looks like some PNVs are being entered with (NR) and others without, and
I dont know when that started.

**Comment by splintermathwood** [comment]
Hi, Splinter

So, there was a guy who wanted to be Ranger Grizzly. His first year he showed up with only Ranger logo’d clothing. ONLY Ranger logo’d clothing. Had it all made before he came to the playa. He brought NO non-Ranger logo’d clothing to the Playa. 

He was not made a Ranger that year for about 50 reasons. Though I was in charge of the Mentor program, I was one of his mentors. Because well, we were worried about him. 

I stepped away from the program and I am pretty sure Grizzly tried again. He (Jonothon Ames) is not in the clubhouse as ever having become a Ranger. 

Do we now track people like this and how?

Thanks for all your work.

Splinter

**Comment by rangerthreepio** [comment]
I've never responded directly to Github before, so we'll see how this goes:


   1. This is a crazy corner case. People who come to the window on playa
   and haven't applied at all are basically shown the door today unless its a
   really special case.
   2. If you don't get accepted into the process by anyone, you wouldn't
   have a clubhouse account unless you set yourself up as an auditor
   3. Any Cadre can keep records on people who aren't in the clubhouse, but
   thats a pretty imperfect system.
   4. I can't think of anyone who has been a repeat issue since I've been a
   VC, who is not in the clubhouse.
   5. This is a topic for the intake team, not this thread about clubhouse
   naming conventions. ;)

**Comment by corprew** [comment]
hey, folks:

> let's not stop with (NR) until such time as someone has gone through and made sure the reports (or other places where a Ranger callsign appears) either show status or are gated by status.

I'd like to make a +1 to that because it makes some RRN report usage easier.  I will add some notes to #172 about our requirement/use because it's become more common in 2019.

/zg

**Comment by rangerthreepio** [comment]
If I understand correctly, the proposed changes would make going back in
and adding (NR) to past prospectives who don't have it moot, right?

Just checking whether the VCs are going to have an action item to correct
the database.

**Comment by mikeburg** [comment]
Implemented by https://github.com/burningmantech/ranger-clubhouse-api/pull/303

**Comment by mikeburg** [comment]
Closing out.
